tests/generator/test_rom.py

```diff
@@ -155,7 +155,7 @@ def test_get_name_puzzle_conditions(self, softfork_height: int) -> None:
             agg_sig_puzzle_amount=[],
             agg_sig_parent_amount=[],
             agg_sig_parent_puzzle=[],
-            flags=ELIGIBLE_FOR_DEDUP,
+            flags=0,
```

Comment:
these flags are now only populated in mempool mode (when the `ANALYZE_SPENDS` flag is set)